src/Compilers/CSharp/Portable/Binder/Binder_NameConflicts.cs

```diff
@@ -111,13 +112,28 @@ protected bool ValidateNameConflictsInScope(Symbol symbol, Location location, st
                     break;
                 }
 
+                // symbols inside local function shadow names in scopes outside local function
```

Comment:
I thought this new behavior would be only for static local functions too.

Do you have some tests with nesting: a local function inside a static local function that shadows type parameters or parameters?
```
void M(int x)
{
   static void local()
   {
       void local2(int x) { } // allowed or not?
  }
}
```

Reply:
Added tests.

---
In reply to: [244605531](https://github.com/dotnet/roslyn/pull/32067#discussion_r244605531) [](ancestors = 244605531)